tests/sentry/integrations/bitbucket_server/testutils.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import absolute_import
+from collections import OrderedDict
+
+EXAMPLE_PRIVATE_KEY = """-----BEGIN RSA PRIVATE KEY-----
```

Comment:
@pierredup Where is this private key from? This looks like a real private key which makes me nervous, even if it's no longer valid. Probably better to make it look faker so no one thinks it's a valid private key :)

Reply:
@scefali I just copied it from [tests/sentry/integrations/jira_server/testutils.py](https://github.com/getsentry/sentry/blob/master/tests/sentry/integrations/jira_server/testutils.py#L3)

Reply:
@pierredup
That’s fine then :)